account-lookup-service/requirements.yaml

```diff
@@ -8,6 +8,10 @@ dependencies:
   version: 8.3.0
   repository: "file://./chart-admin"
   condition: account-lookup-service-admin.enabled
+- name: als-oracle-pathfinder
```

Comment:
Ensure that you update the `account-lookup-service/chart.yaml` appVersion field to include the als-oracle-pathfinder